MantidPlot/src/ProjectRecoveryGUIs/ProjectRecoveryModel.cpp

```diff
@@ -216,4 +216,17 @@ std::string ProjectRecoveryModel::decideLastCheckpoint() {
   auto mostRecentCheckpointPath = mostRecentCheckpoints.back();
   return mostRecentCheckpointPath.directory(mostRecentCheckpointPath.depth() -
                                             1);
+}
+
+int ProjectRecoveryModel::getNumberOfCheckpoints() {
```

Comment:
Could this function be marked as `static` since it does not seem to need anything from `this`?

Reply:
:+1: